examples/basic/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "do_token" {
```

Comment:
Add dot at end of dsecription.
Add `default = null` - so provider will fallback to environment variables if variable is not set.